src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+const text = [...population].map(data => (
```

Comment:
```suggestion
const text = population.map(data => (
```

Reply:
it won't work
![image](https://user-images.githubusercontent.com/79811705/199608613-07357aad-d28e-40fb-9bb0-0ab443cb1b34.png)
![image](https://user-images.githubusercontent.com/79811705/199608700-873f6eb7-926f-4700-947c-763db78c3181.png)

Reply:
it doesn't work because querySelectorAll returns the list of elements, which you need to convert in array